elements/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -170,4 +167,22 @@ export class TransactionPool {
 	}
 
 	private async _reorganize(): Promise<void> {}
+
+	private _getStatus(
+		txResponse: ReadonlyArray<TransactionResponse>,
+	): TransactionStatus {
+		const txResponseErrors = txResponse[0].errors;
```

Comment:
if `errors` is 0 and status.ok, it should be processable

Reply:
In AddTrx case, we will only get either Invalid or unProcessable trx. Only job can promote and make a trx processable. isn't it?